py/torch_migraphx/fx/converters/acc_ops_converters.py

```diff
@@ -1135,6 +1135,31 @@ def acc_ops_embedding(mgx_module, node, args, kwargs):
         mgx_module.add_instruction(migraphx.op('gather', axis=0),
                                    [weight.instr_ref, inp.instr_ref]))
 
+@migraphx_converter(acc_ops.gather)
+def acc_ops_gather(mgx_module, node, args, kwargs):
+    inp = kwargs['input']
+    dim = kwargs['dim']
+    index = kwargs['index']
+
+    assert not inp.is_quantized() and not index.is_quantized()
+    
+    dimensions = index.shape().lens()
```

Comment:
```suggestion
    index_lens = index.shape().lens()
```
A little nitpick but its a little confusing to keep track of too many similar variables "dimension", "dim", "dims"